docs/DATA_OVERHAUL_SPLIT.md

```
@@ -4,6 +4,12 @@ PR #177 is the backup/source branch for this migration. Do not keep expanding
 that branch. Split it into small PRs that can be reviewed and validated
 independently.
 
+Each split PR should start by extracting the relevant implementation from PR
+#177, then adapting only what is necessary for the current branch. Do not
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)